databases/schemas/modlog-fts.sql

```diff
@@ -0,0 +1,33 @@
+SELECT load_extension("native/fts_id_tokenizer.o");
+
+CREATE TABLE IF NOT EXISTS modlog (
```

Comment:
I see no reason to duplicate this definition, you can just have the `load_extension` + `create virtual table` in a separate file that is run after `modlog.sql`